proto/reaction.proto

```diff
@@ -389,19 +465,19 @@ message TemperatureConditions {
 }
 
 message PressureConditions {
+  // TODO Consider renaming given adjustment in meaning
   message PressureControl {
     enum PressureControlType {
       UNSPECIFIED = 0;
       CUSTOM = 1;
       AMBIENT = 2;
-      BALLOON = 3;
-      SEALED = 4; // Fully sealed vessel (e.g., microwave vial).
-      SEPTUM_WITH_NEEDLE = 5;  // Slight positive pressure maintained
-      RELEASEVALVE = 6;
-      BPR = 7; // Back pressure regulator, as used in flow synthesis.
+      SLIGHT_POSITIVE = 3;  // E.g., with a gas line inlet.
```

Comment:
Just `POSITIVE`? Maybe group closer to `PRESSURIZED` and add a comment to help distinguish between these options?

Reply:
This is really referring to a single (extremely common) setup, written in the comment, where a gas line with a needle punctures a septum or cap on the vessel. So, while there is a "slight-positive" pressure, there's typically no intention of actually "pressurizing" the flask.

Reply:
+1 that `SLIGHT_POSITIVE` means something distinct from `POSITIVE`. In the common case that Mike describes, the setpoint would not be specified. `SLIGHT_POSITIVE` usually implies 1-5 psi